### PR TITLE
Add Cloud SQL connection pooling sample app.

### PR DESCRIPTION
**Pull request by jsimonweb**

### CODE REVIEW

File: cloudsql/appengine/flexible/Startup.cs [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+            connectionString.MaximumPoolSize = 5;
+            // [END cloud_sql_mysql_max_connections]
+            // [START cloud_sql_mysql_connection_timeout]
+            connectionString.ConnectionTimeout = 15;
+            // [END cloud_sql_mysql_connection_timeout]
+            // [START cloud_sql_mysql_connection_lifetime]
+            connectionString.ConnectionLifeTime = 300;
+            // [END cloud_sql_mysql_connection_lifetime]
+            // [END_EXCLUDE]
+            DbConnection connection =
```
Comment (kurtisvg) [inline cloudsql/appengine/flexible/Startup.cs]: Is this a pool, or is it a connection? 

Can we add either some additional comments or perhaps a link explaining how connections are reused, and what the best practices around closing or retrieving is?

File: cloudsql/appengine/flexible/Filters/DbExceptionFilter.cs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+ * the License.
+ */
+using System.Data.Common;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+
+public class DbExceptionFilterAttribute : ExceptionFilterAttribute
```
Comment (SurferJeffAtGoogle) [inline cloudsql/appengine/flexible/Filters/DbExceptionFilter.cs]: Add a class comment, describing the purpose this filter serves.

File: cloudsql/appengine/flexible/Filters/DbExceptionFilter.cs [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        IModelMetadataProvider modelMetadataProvider)
+    {
+        _hostingEnvironment = hostingEnvironment;
+        _modelMetadataProvider = modelMetadataProvider;
+    }
+
+    public override void OnException(ExceptionContext context)
+    {
+        var dbException = context.Exception as DbException;
+        if (!_hostingEnvironment.IsDevelopment() || null == dbException)
```
Comment (SurferJeffAtGoogle) [inline cloudsql/appengine/flexible/Filters/DbExceptionFilter.cs]: Why swallow the exception?

File: cloudsql/mysql/Startup.cs [hunk 4]
```diff
@@ -0,0 +1,242 @@
...
+            }
+            connection.Open();
+            using (var createTableCommand = connection.CreateCommand())
+            {
+                createTableCommand.CommandText = @"
+                    CREATE TABLE IF NOT EXISTS
+                    votes(
+                        vote_id SERIAL NOT NULL,
+                        time_cast timestamp NOT NULL,
+                        candidate CHAR(6) NOT NULL,
```
Comment (kurtisvg) [inline cloudsql/mysql/Startup.cs]: I (think) there was a small issue with char and varchar causing an issue between MySQL and Postgres. 

For [MySQL](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/6ec5bed56607605a443037c26e659c1f1282928e/cloud-sql/mysql/sqlalchemy/main.py#L92) we used CHAR
For [Postgres](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/cloud-sql/postgres/sqlalchemy/main.py#L92) we used VARCHAR

**Pull request merged**